### PR TITLE
feat: add variant options with new colors

### PR DESCRIPTION
**Pull request by r-mulder**
In this PR I've updated the divider to support the new colors and have the options to choose which one to use. 

Different colors are being used when using the new variant prop.

### CODE REVIEW

**Review by ferreroltd — APPROVED** [review]

File: packages/libs/react-ui/src/components/Divider/Divider.css.ts [hunk 1]
```diff
@@ -1,15 +1,34 @@
...
-    height: tokens.kda.foundation.border.width.hairline,
+export const dividerClass = recipe({
+  base: [
+    atoms({
+      width: '100%',
+      marginBlock: 'lg',
+      border: 'none',
+    }),
+    {
+      height: tokens.kda.foundation.border.width.hairline,
```
Comment (ferreroltd) [inline packages/libs/react-ui/src/components/Divider/Divider.css.ts:13]: do we prefer tokens over token() then

File: .changeset/shiny-crews-sleep.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@kadena/react-ui": minor
+---
+
+Add variant option for different colors
```
Comment (eileenmguo) [inline .changeset/shiny-crews-sleep.md:5]: I don't think this is descriptive enough for someone reading release docs. Maybe something like "Added a new variant to apply different colors to the Divider component"

File: packages/libs/react-ui/src/components/Divider/Divider.css.ts [hunk 3]
```diff
@@ -1,15 +1,35 @@
...
+        backgroundColor: token('color.border.base.default'),
+      },
+      bold: {
+        backgroundColor: token('color.border.base.bold'),
+        boldest: {
+          backgroundColor: token('color.border.base.boldest'),
+        },
+        highContrast: {
+          backgroundColor: token('color.border.base.high-contrast'),
+        },
```
Comment (eileenmguo) [inline packages/libs/react-ui/src/components/Divider/Divider.css.ts:31]: I feel like these are at the wrong level?

Comment (r-mulder) [reply packages/libs/react-ui/src/components/Divider/Divider.css.ts:31]: You mean that they should be in the atoms?

File: .changeset/shiny-crews-sleep.md (hunk 2 above)
Comment (r-mulder) [reply .changeset/shiny-crews-sleep.md:5]: Agreed, changed.

File: packages/libs/react-ui/src/components/Divider/Divider.css.ts (hunk 3 above)
Comment (r-mulder) [reply packages/libs/react-ui/src/components/Divider/Divider.css.ts:31]: nvm, I just noticed that I make a mistake while resolving Sams comment.

**Pull request merged**